### PR TITLE
ingress: add dummy HTTPProxies

### PR DESCRIPTION
**Pull request by dulltz**
This PR is to prevent envoy crush loop.
See: https://github.com/cybozu-go/neco/issues/1002

### CODE REVIEW

File: ingress/base/template/httpproxy.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+spec:
+  virtualhost:
+    fqdn: dummy.ingress.dummy
+    tls:
+      secretName: dummy-httpproxy-tls
+  routes:
+    - conditions:
+        - prefix: /
+      services:
+        - name: dummy-tls
```
Comment (ysksuzuki) [inline ingress/base/template/httpproxy.yaml:18]: This service name should be `dummy` not `dummy-tls`, shouldn't it?

File: ingress/base/bastion/httpproxy.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+apiVersion: projectcontour.io/v1
+kind: HTTPProxy
+metadata:
+  name: dummy
+  namespace: ingress
```
Comment (zoetrope) [inline ingress/base/bastion/httpproxy.yaml:5]: Is the namespace correct?
`ingress` namespace doesn't exist.
It should be `ingress-bastion`.

Comment (zoetrope) [reply ingress/base/bastion/httpproxy.yaml:5]: Sorry, I misunderstood.
The namespace will be overwrite in kustomization.yaml.

File: ingress/base/template/httpproxy.yaml (hunk 1 above)
Comment (kfyharukz) [reply ingress/base/template/httpproxy.yaml]: You're right. I corrected it!

**Pull request merged**